pkg/controller/metrics/diverters/MetricsDispatcher.go

```diff
@@ -0,0 +1,55 @@
+package diverters
+
+import (
+	"fmt"
+	"github.com/opensds/opensds/pkg/model"
+)
+
+// A buffered channel that we can send work requests on.
+var MetricsQueue = make(chan model.MetricSpec, 100)
+
+func SendMetricToRegisteredSenders(metrics model.MetricSpec) {
+
+	// Push the work onto the queue.
+	MetricsQueue <- metrics
+	fmt.Println("Send metrics request queued")
```

Comment:
Please use log to print and store the message, you can import `log "github.com/golang/glog"`.

Reply:
updated in all places.